### DISCUSSION TITLE
Card: Rupert Murdoch

### DISCUSSION THREAD

**Discussion by RebelusQuo** [RebelusQuo/the-wall-is-down#11] [Ideas]
I'd like to rely on the "do as much as possible" principle, skipping "up to". I also think it's a good idea to make it explicit in the card text that the limitations has to be followed, or ignored for e.g. Petrodollars (discussed in a separate topic). Suggestion is thus as follows:

> US player vie for Supremacy over two Public Opinion Slots, under Vetoes and Second Epoch limitations.

**Comment by rodrigoSantamaria** [comment]
Ok, we can rely on that and remove "up to". The 'up to' clauses are mostly to allow to play the card if you cannot use all the influence/supremacy/etc.
I'd also remove the 'under vetoes and second epoch limitations', and make it explicit only when you can overcome that limitations, but I trust you on this, whenever you think is ok, is ok to me.

**Comment by RebelusQuo** [comment]
What we can do is to keep this as a reminder, in a pair of parenthesis, while the other cards that ignore them have "first class" card text formulations?

**Comment by rodrigoSantamaria** [comment]
Yes, good solution!

**Comment by RebelusQuo** [comment]
Suggestion:
> US player vie for Supremacy over two Public Opinion Slots (following normal Vetoes and Second Epoch limitations).

**Reply by rodrigoSantamaria** [reply]
Ok, perfect!
